README.md

```diff
@@ -298,6 +298,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [MDX Notes](https://github.com/maqi1520/mdx-notes/tree/tauri-app) - Versatile WeChat typesetting editor and cross-platform Markdown note-taking software.
 - [Noor](https://noor.to/) ![closed source] - Chat app for high-performance teams. Designed for uninterrupted deep work and rapid collaboration.
 - [Notedown](https://github.com/ruralad/notedown) - Minimal, local-first cross-platform note taking app.
+- [Notpad](https://github.com/Muhammed-Rahif/Notpad) - The ordinary notepad that everyone know, but it's rich!
```

Comment:
```suggestion
- [Notpad](https://github.com/Muhammed-Rahif/Notpad) - The ordinary notepad that everyone knows, but it's rich!
```

Would you mind a more, i don't know, "objective" description? The other apps are described more "flatly" and i'd prefer to keep it in line tbh. Also, i think not everyone understands what "it's rich!" actually entails?

Reply:
What about "Cross-platform rich text editor with a notepad interface, enhanced with advanced features beyond standard notepad."

Reply:
yeah, that would work for me :) thanks

Reply:
Let me change.